dmap.js

```diff
@@ -36,28 +34,18 @@ lib._walk = async (dmap, path, register, ctx, trace) => {
     }
 
     const step = path[0]
-    const rune = step[1][0]
-    const name = step[1][1]
+    const rune = step[children][0][text]
```

Comment:
More of this logic should live in parse/postparse, at this step you should have a list of meaningful objects and not need to know indices of nested lists